src/store/modules/facility/actions.ts

```diff
@@ -177,11 +177,19 @@ const actions: ActionTree<FacilityState, RootState> = {
     commit(types.FACILITY_CURRENT_UPDATED, facility)
   },
 
-  async fetchCurrentFacility({ commit, dispatch, state }, payload) {
+  async fetchCurrentFacility({ commit, dispatch, rootGetters, state }, payload) {
     // checking that if the list contains basic information for facility then not fetching the same information again
     const cachedFacilities = JSON.parse(JSON.stringify(state.facilities.list))
     const current = cachedFacilities.find((facility: any) => facility.facilityId === payload.facilityId)
     if(current?.facilityId && !payload.skipState) {
+      // As inventory channels are fetched while fetching additional facility info
+      // But here we already have additional facility info, so just getting and adding inventory groups to current.
+      const inventoryGroups = rootGetters['util/getInventoryGroups'];
+      inventoryGroups.forEach((group: any) => {
+        const isChecked = (current.facilityGroupInfo?.some((facilityGroup: any) => facilityGroup?.facilityGroupId === group.facilityGroupId))
+        group.isChecked = isChecked ? isChecked : false;
```

Comment:
```suggestion
        group.isChecked = (current.facilityGroupInfo?.some((facilityGroup: any) => facilityGroup?.facilityGroupId === group.facilityGroupId))
```
There is no need to have extra check.

Reply:
This has been handled in https://github.com/hotwax/facilities/pull/182